_data/events_gdcr2019/duesseldorf.json

```diff
@@ -0,0 +1,40 @@
+{
+  "title": "GDCR2019 SWK D'dorf Vorwerk",
+  "url": "https://www.meetup.com/de-DE/Softwerkskammer-Dusseldorf/",
+  "moderators": [
+    {
+      "name": "Birgit Kratz",
+      "url": "https://www.xing.com/profile/Birgit_Kratz2"
+    },
+    {
+      "name": "Tim Riemer",
+      "url": "https://twitter.com/zordan_f"
+    },
+    {
+      "name": "Michael Arends",
+      "url": "https://corporate.vorwerk.de/home/"
```

Comment:
This is the same link of sponsor. Please, if Michael Arends doesn't have a personal page (Twitter, Linkedin, Xing, ...), remove `url` property in this case. Thanks in advance!

Reply:
Hi Rachel, thanks for the hint. I removed the url and created a new pull request:
https://github.com/coderetreat/coderetreat.org/pull/556

Reply:
Great!! Thanks @ufh :tada: